features/head.tsx

```
@@ -239,29 +399,102 @@ function HeadElement({
+      console.error(`[Bunext Head] Error reading CSS file ${filePath}:`, error);
+      return '';
+    }
   };
 
+  const SSRStyle = useMemo(() => {
+    const stylesElements = typeof window === "undefined" && getPaths().map((cssPath, i) => {
+      const cssContent = getStringData(cssPath);
+      return cssContent ? (
+        <style
+          key={i}
+          dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)